picard/log.py

```diff
@@ -71,6 +73,24 @@ def get_effective_level():
     return main_logger.getEffectiveLevel()
 
 
+def set_verbosity(level, save_to_config=False):
```

Comment:
I think we don't need both `set_level` and `set_verbosity`. Having both makes for a confusing API where it is unclear which one to use.

The change here to extend the function to handle errors and allow config saving makes sense. But I think we could either extend `set_level` with the functionality or replace it with `set_verbosity` if you prefer that name. No strong opinion either way, maybe I slightly prefer `set_verbosity` to be slightly more specific.

Reply:
I got rid of `set_level()` in https://github.com/metabrainz/picard/pull/2653/commits/c1e7b26d3bf95e93e79023330da043272c001417